### ISSUE TITLE
Repos seem to have crossed paths

### ISSUE THREAD

**Issue by Peter-Easterbrook** [desktop/desktop#13604]
### Describe the bug
Peter-Easterbrook/PEWDResume no longer shows on GitHub Desktop, I have to open it manually.
When committing changes via vscode terminal I got this: 
![Screenshot 2021-12-28 080610](https://user-images.githubusercontent.com/59874288/147538353-59dcbde3-18cc-40dd-8140-27105c574dbb.png)

Looking in GitHub desktop, it seems it's trying to push the PEWDResume changes to the crwn-clothing repo

### Version & OS

GitHub desktop version: 2.9.6 (x64)
Windows 10 pro

### Steps to reproduce the behavior

1. Attempt to commit via vscode terminal to PEWDResume
2. See error above

### Expected behavior

Expect commit to PEWDResume

### Actual behavior

As above

**Comment by steveward** [comment]
@Peter-Easterbrook thanks for the report. Can you share the output of `git remote -v` from the command line for the repository? That will let us know if the `origin` remote of your repository is misconfigured.

**Comment by Peter-Easterbrook** [comment]
Hey Steve,
thanks for getting back to me about that.
In the meantime, I already looked into the .git config file and saw that
somehow the repo origins had been switched, at least the PEWDResume  repo
address had been switched for the crwn-clothing-2021 address.
So I just entered the correct address, pushed and it cleared itself up
though it's a curious thing to happen as, obviously, I don't interfere with
.git files.

Best regards,
Peter.
